examples/viewer.py

```diff
@@ -412,12 +478,18 @@ def key_press_event(self, event: Application.KeyEvent) -> None:
         elif key == pressed.H:
             self.print_help_text()
 
+        elif key == pressed.ONE:
```

Comment:
A bit of background, in `viewer.cpp`, each individual Habitat RE gradually added custom debugging functionality over time, to the point where that tool has a baffling huge set of hotkeys and is really not user-friendly. I would love to see us not make that mistake again.

Maybe my concrete suggestion here is to not add this hotkey. I suspect @0mdc is the only person that will ever use this functionality (you could add this code locally or in a branch).